DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditDelegateCourseAdminField.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+@model EditDelegateCourseAdminFieldViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? $"Error: Edit {Model.PromptText} field for course" : $"Edit {Model.PromptText} field for course";
+
+  const string blankRadioId = "leave-blank";
+
+  var routeParamsForBackLink = new Dictionary<string, string?>();
+  string cancelLinkController;
+
+  // TODO: HEEDLS-862 Account for all possible access routes in back link
+  if (Model.AccessedVia.Equals(DelegateAccessRoute.CourseDelegates)) {
+    routeParamsForBackLink.Add("accessedVia", Model.AccessedVia.ToString());
+    routeParamsForBackLink.Add("progressId", Model.ProgressId.ToString());
+    routeParamsForBackLink.Add("returnPage", Model.ReturnPage.ToString());
+    cancelLinkController = "DelegateProgress";
+  } else {
+    routeParamsForBackLink.Add("delegateId", Model.DelegateId.ToString());
+    cancelLinkController = "ViewDelegate";
+  }
+}
+
+@*TODO: HEEDLS-862 - Change these breadcrumbs to account for all access routes*@
+
+@section NavBreadcrumbs {
+  @if (Model.AccessedVia.Equals(DelegateAccessRoute.CourseDelegates)) {
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_CourseDelegatesProgressBreadcrumbs.cshtml" model="(Model.CustomisationId, Model.ProgressId)" />
+  } else {
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_ViewDelegateProgressBreadcrumbs.cshtml" model="(Model.DelegateId, Model.ProgressId)" />
+  }
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(@Model.Answer)}" />
+    }
+
+    <h1 class="nhsuk-heading-xl word-break">Edit @Model.PromptText field for @Model.CourseName</h1>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-quarter nhsuk-body-l">
+        Delegate
+      </div>
+      <div class="nhsuk-grid-column-three-quarters nhsuk-body-l nhsuk-u-font-weight-bold word-break">
+        @Model.DelegateName
+      </div>
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-half">
+
+        <form class="nhsuk-u-margin-bottom-3" method="post" novalidate>
+          <input type="hidden" asp-for="DelegateId" />
+          <input type="hidden" asp-for="ProgressId" />
+          <input type="hidden" asp-for="ReturnPage" />
+
+          @if (Model.Options.Count > 0) {
+            <div class="nhsuk-radios nhsuk-u-margin-bottom-4">
+              @foreach (var radio in Model.Radios) {
+                var isBlankOption = radio.Value == string.Empty;
+                <div class="nhsuk-radios__item nhsuk-u-margin-bottom-3 @(isBlankOption ? "nhsuk-u-margin-top-6" : "")">
+                  <input class="nhsuk-radios__input"

```

Comment:
We've decided to separate the "Leave blank" radio button from the rest to make it clear it's not an ordinary option
![localhost_5001_TrackingSystem_Delegates_CourseDelegates_DelegateProgress_2803_EditAdminField_1](https://user-images.githubusercontent.com/70278044/160791498-94df79d7-2cbc-49ba-a824-f61678772662.png)